### PR TITLE
Remove redirect of `test:mocha:multireport` to `test:mocha`, which are no longer necessary

### PR DESCRIPTION
**Pull request by curtisman**
After #16913, the separate `*:report` script for mocha test is no longer necessary. clean those up.

Also, disable `test:mocha` script in `devtools-view` and `devtools-browser-extension` as they are broken (and wasn't ran in the CI previously because they don't have `test:mocha:multireport` script.

### CODE REVIEW

**Comment by Josmithr** [comment]
ADO work items tracking fixing the tests and associated scripts in `devtools-view` and `devtools-browser-extension`: [ADO#5413](https://dev.azure.com/fluidframework/internal/_workitems/edit/5413) and [ADO#5414](https://dev.azure.com/fluidframework/internal/_workitems/edit/5414).

File: package.json [hunk 1]
```diff
@@ -26,8 +26,8 @@
 		"ci:build": "fluid-build --task ci:build",
 		"ci:test:jest": "npm run test:jest:report",
 		"ci:test:jest:coverage": "c8 --no-clean npm run test:jest:report",
-		"ci:test:mocha": "npm run test:mocha:report",
-		"ci:test:mocha:coverage": "c8 --no-clean npm run test:mocha:report",
+		"ci:test:mocha": "npm run test:mocha",
+		"ci:test:mocha:coverage": "c8 --no-clean npm run test:mocha -- --timeout 4s",
```
Comment (Josmithr) [inline package.json:30]: Is this right? Should this not be calling `test:mocha:coverage`?

**Review by Josmithr — APPROVED** [review]
This is a huge improvement. Thank you!!

File: package.json (hunk 1 above)
Comment (curtisman) [reply package.json:30]: `c8` executable in front will inject coverage collection to all child process.   It is faster than calling `c8` on each individual command the last I checked.

**Pull request merged**